suite-common/connect-init/src/bindConnectActionsListeners.ts

```diff
@@ -0,0 +1,39 @@
+import TrezorConnect, {
```

Comment:
Can you move this back to thunk and get rid of functions, eg use directly dispatches.

Reply:
https://github.com/trezor/trezor-suite/pull/5760/commits/979f44f2e30665c851548bc75ae31ae9d9bbba9b